pkg/engine/rule/r0002_unexpected_file_access.go

```diff
@@ -103,6 +126,26 @@ func (rule *R0002UnexpectedFileAccess) ProcessEvent(eventType tracing.EventType,
 	}
 }
 
+func getMountPaths(podName string, namespace string, containerID string, engineAccess EngineAccess) ([]string, error) {
+	podSpec, err := engineAccess.GetPodSpec(podName, namespace, containerID)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get pod spec: %v", err)
+	}
+
+	mountPaths := []string{}
+	for _, container := range podSpec.Containers {
+		for _, volumeMount := range container.VolumeMounts {
+			mountPaths = append(mountPaths, volumeMount.MountPath)
+		}
+	}
+
+	return mountPaths, nil
+}
```

Comment:
Something is does not add up here. We do not take into account here in which container the event belongs to. We should only return here the mounts of the container in which the event happened. Something like this

```
mountPaths := []string{}
	for _, container := range podSpec.Containers {
                if container.name == containerName {
		  for _, volumeMount := range container.VolumeMounts {
			mountPaths = append(mountPaths, volumeMount.MountPath)
		  }
                }
	}
```